packages/story-editor/src/components/canvas/karma/selection.karma.js

```diff
@@ -119,29 +119,6 @@ describe('CUJ: Creator can Transform an Element: Selection integration', () => {
     await fixture.snapshot();
   });
 
-  it('should show the selection on top of page menu', async () => {
```

Comment:
@barklund FYI I removed this test since it seems to be failing now after the workspace changes (the element gets moved out of the page and deleted), however, what it was testing before is not relevant anymore at all due to moving the page menu.